test_pool/exerciser/operating_system/test_os_e004.c

```diff
@@ -27,6 +27,7 @@
 #define TEST_NUM   (ACS_EXERCISER_TEST_NUM_BASE + 4)
 #define TEST_RULE  "PCI_MSI_2,ITS_DEV_6"
 #define TEST_DESC  "MSI(-X) triggers intr with unique ID  "
+#define LPI_MINID  8192
```

Comment:
We already have ARM_LPI_MINID defined in acs_gic_support.h, we can use the same here.